src/Microsoft.Data.Analysis/PrimitiveDataFrameColumn.cs

```diff
@@ -328,6 +328,16 @@ public override string ToString()
             return $"{Name}: {_columnContainer.ToString()}";
         }
 
+        public new PrimitiveDataFrameColumnRollingWindow<T> Rolling(int windowSize)
```

Comment:
I wonder if inherit doc works here

Reply:
It won't work on `new` members. Only override and interface implementations.

---
In reply to: [379294117](https://github.com/dotnet/corefxlab/pull/2827#discussion_r379294117) [](ancestors = 379294117)